client/src/components/Debts/DebtList.js

```diff
@@ -0,0 +1,146 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import DebtForm from "./DebtForm";
+
+const emptyDebt = {
+  which_bill: 0,
+  subtotal: 0,
+  how_much: 0,
+  who_owes: 0,
+  debt_paid_up: 0,
```

Comment:
❓ should this be `0` or `null`?

Reply:
These should be null. There are some values in BillForm which are set to 0 right now because I haven't added in the calculator functionality to calculate certain values yet. These changes can be seen in "Forms" (did there instead, because the empty consts were made into their own components).

Reply:
Debts have been made null or "". Bills are null, 0, or "". The zeros will be replaced when the math works.